.golangci.yml

```diff
@@ -5,10 +5,7 @@ linters:
     - staticcheck
     - unused
     - gosimple
-    - structcheck
-    - varcheck
     - ineffassign
-    - deadcode
```

Comment:
これ削除しないと通らない？

Reply:
これら、WARN: deprecatedになってて使われてなかったんですよね
削除しなくても動くんですけど…どうしましょうかね

Reply:
これか、commit messageが無で勘違いしてた

>   level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused."
  level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused."
  level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused."

https://github.com/traPtitech/traQ/runs/8219646784?check_suite_focus=true

なら良さそう :+1:

Reply:
commit messageもうちょい書くべきだというのは、それはそう過ぎて申し訳ない
ありがとうございます